guides/common/modules/ref_network-interface-configuration-options.adoc

```diff
@@ -0,0 +1,38 @@
+[id="network_interface_configuration_options"]
+= Network interface configuration options
+
+When adding a network interface, you need to specify several configuration options. The following lists provide information on the relevant options for the different types of interfaces.
+
+.Physical interface settings
+
+*Device Identifier*:: The identifier is used to specify this physical interface when creating bonded interfaces, VLANs, and aliases.
+
+*DNS name*:: {Project} saves this name in the {SmartProxyServer} associated with the selected domain (the "DNS A" field) and {SmartProxyServer} associated with the selected subnet (the "DNS PTR" field).
+A single host can therefore have several DNS entries.
+
+*Domain*:: Specifies the domain associated with the network interface. To create and manage domains, navigate to *Infrastructure* > *Domains*.
+
+*Subnet*:: Specifies the subnet associated with the network interface. To create and manage subnets, navigate to *Infrastructure* > *Subnets*.
+
+*Managed* interface:: If the interface is managed, configuration is pulled from the associated {SmartProxyServer} during provisioning, and DNS and DHCP entries are created.
+If using Kickstart provisioning, a configuration file is automatically created for the interface.
```

Comment:
```suggestion
If using installer-based provisioning, a configuration file is automatically created for the interface.
```
Kickstart is installer configuration specific to EL, but I think it applies to any OS installer. I'd go with a more generic term to make it more upstream friendly.

Reply:
@maximiliankolb Can you or someone from your team confirm it applies to other OSs as well?

Reply:
In the context of Foreman, installer-based provisioning should be considered automated. Anaconda+Kickstart is for EL, but Debian/Ubuntu uses a different installer+config. But Foreman generates configuration for Debian/Ubuntu as well, it just isn't called Kickstart.
Then perhaps a better term would be:
```suggestion
If you use provisioning with installer automation, a configuration file is automatically created for the interface.
```

Reply:
@stejskalleos Would that ^^ work?

Reply:
If, in this context, we expect the installer workflow always to be automated, then LGTM.

Reply:
Using a OS-neutral term would be appreciated, yes.